scripts/lib/update_acme_tests.py

```diff
@@ -42,8 +42,8 @@
                              "ERI.f09_g16.X",
                              "ERIO.f09_g16.X",
                              "SEQ_Ln9.f19_g16_rx1.A",
-                             "ERS.ne30_g16_rx1.A",
-                             "ERS_N2.f19_g16_rx1.A",
+                             "IRT.ne30_g16_rx1.A",
```

Comment:
Now we don't have any ERS tests in cime_developer. That can't be good, right?

Reply:
The IRT is an expansion on ERS - it does the same thing + more.

Reply:
I agree with @jgfouca that there should be at least one ERS test in cime_developer, partly to make sure that the ERS test itself doesn't get broken.